CDCgov/NEDSS-Modernization
# CNFT2-1185 split create question

## Description

Due to the complexity of having a single endpoint for creating a question, this PR splits this into multiple routes.
1. `/api/v1/questions/text`
2. `/api/v1/questions/numeric`
3.  `/api/v1/questions/date`
4. `/api/v1/questions/coded`

There was also a critical vulnerability found in Kafka. The only place that we were using Kafka in Pagebuilder was the create question API. This PR also removes Kafka. 

## Tickets

* [CNFT2-1185](https://cdc-nbs.atlassian.net/browse/CNFT2-1185)

## Steps to verify

1. Using swagger, create a question of each type. 
1. Verify the new questions appear in the classic question list

[CNFT2-1185]: https://cdc-nbs.atlassian.net/browse/CNFT2-1185?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
